### PR TITLE
chore(docs): update docs with configs

### PR DESCRIPTION
**Pull request by adamdehaven**
### Summary

Update Getting Started docs with up-to-date install and usage instructions.

#### Changes made:
* Add instructions for `transpileDependencies` and PostCSS preserve properties.
* Updated jsfiddle (tied to Darren's account) to a new CodePen that also properly imports `@kongponents/styles`

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -41,7 +106,14 @@ export default {
 
 ## Without Bundle System
 
-You can also install Kongponents into a project where there is no build system
-as long as Vue is included. Each Kongponent is packaged as a `umd.js` file, so as long as you have loaded Vue in your project the Kongponent will work as intended. You will still need to [install the base styles package](./style-guide/usage.md).
+You can also use Kongponents in a project where there is no build system as long as Vue is included on the page. Each Kongponent is packaged as a `umd.js` file, so as long as you have loaded Vue in your project the Kongponent will work as intended.
```
Comment (chrisvdp) [inline docs/README.md:109]: do you think its worth linking to https://cli.vuejs.org/guide/build-targets.html#library ?

Comment (adamdehaven) [reply docs/README.md]: 👍🏼  added link

File: docs/README.md [hunk 2]
```diff
@@ -7,28 +7,93 @@ next: false
...
+// vue.config.js
+
+module.exports = {
+  transpileDependencies: [
+    /@kongponents\/.*/
+  ]
+}
+```
+
+If your project does not have a `vue.config.js` file and instead uses webpack config files, you can add a loadcer rule (for example, for `babel-loader`) similar to the following (only showing the relevant entries)
```
Comment (kaiarrowood) [inline docs/README.md:36]:
```suggestion
If your project does not have a `vue.config.js` file and instead uses webpack config files, you can add a loader rule (for example, for `babel-loader`) similar to the following (only showing the relevant entries)
```

**Review by kaiarrowood — COMMENTED** [review]
This is great @adamdehaven , only one small nit.

**Review by kaiarrowood — APPROVED** [review]

File: docs/README.md (hunk 2 above)
Comment (adamdehaven) [reply docs/README.md]: Fixed 👍🏼

**Pull request merged**